data/init.sql

```diff
@@ -65,7 +65,7 @@ CREATE TABLE IF NOT EXISTS marked_entities (
 	id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
 	title VARCHAR(50),
 	description TEXT,
-	course_offering_id INT(11) UNSIGNED NOT NULL,
+	course_section_id INT(11) UNSIGNED NOT NULL,
 	is_group_work BOOLEAN,
```

Comment:
🤔 hmm should marked entities be created for the entire class instead of individual sections? @drouba @chris7neves what do you guys think

Reply:
I agree, marked entities should be for a whole course, not the sections

Reply:
Yeah, I think they should link to the course as well